include/uv.h

```diff
@@ -1914,17 +1914,17 @@ struct uv_loop_s {
 UV_EXTERN void* uv_loop_get_data(const uv_loop_t*);
 UV_EXTERN void uv_loop_set_data(uv_loop_t*, void* data);
 
-/* String utilities needed internally for dealing with Windows. */
-size_t uv_utf16_length_as_wtf8(const uint16_t* utf16,
-                               ssize_t utf16_len);
-int uv_utf16_to_wtf8(const uint16_t* utf16,
-                     ssize_t utf16_len,
-                     char** wtf8_ptr,
-                     size_t* wtf8_len_ptr);
-ssize_t uv_wtf8_length_as_utf16(const char* wtf8);
-void uv_wtf8_to_utf16(const char* wtf8,
-                      uint16_t* utf16,
-                      size_t utf16_len);
+/* Path munging utilities needed for dealing with Windows. */
```

Comment:
FWIW, they are not for path munging, but for correctly interacting with any and all Windows Unicode APIs

Reply:
Fair enough. Is your suggestion s/Path munging/Unicode/?

Reply:
yes, that would be my recommendation